tls/s2n_client_hello.h

```diff
@@ -43,6 +43,14 @@ struct s2n_client_hello {
      * issues a hello retry.
      */
     unsigned int parsed : 1;
+
+    /*
+     * SSLv2 ClientHellos have a different format.
+     * Cipher suites are each three bytes instead of two.
+     * And due to how s2n-tls parses the record,
+     * the raw_message will not contain the protocol version.
+     */
+    unsigned int sslv2 : 1;
```

Comment:
Isn't this information already available with the get_client_hello_protocol_version() API? Note --> I assume that function gets the protocol version advertised in the Client Hello, but you might want to verify that since I tried to look at the logic of how it is set and got confused.

Reply:
https://github.com/aws/s2n-tls/pull/3815#discussion_r1100608670